### PR TITLE
Add Laravel 11.0

### PR DESCRIPTION
**Pull request by joanhey**
This version is faster than v10

### CODE REVIEW

**Comment by myaaghubi** [comment]
Nope, with my measurements it is a lot slower than `v10.2`
```

|-------------------|------------------------:|-------------:|----------:|-------------:|
|framework          |requests per second (rps)|relative (rps)|peak memory|relative (mem)|
|-------------------|------------------------:|-------------:|----------:|-------------:|
 > opcache on:
|laravel-11.0       |                62.44 (-)|           1.0|   3.08 (-)|           1.0|
 > opcache off:
|laravel-11.0       |                30.84 (-)|           1.0|  18.98 (-)|           1.0|
```

File: config [hunk 1]
```diff
@@ -24,6 +24,7 @@ fatfree-3.8
 fuelphp-1.9
 kumbia-1.2
 laravel-10.2
```
Comment (myaaghubi) [inline config:26]: There is a conflict here with `v10.3`

**Review by myaaghubi — CHANGES_REQUESTED** [review]
Plz check out for conflict

File: config (hunk 1 above)
Comment (joanhey) [reply config]: Please, try to merge from older to newer PR.
At the bottom are the older PRs.

Comment (myaaghubi) [reply config]: Owh

**Comment by joanhey** [comment]
My comment was a call to attention !!   :)

**Comment by myaaghubi** [comment]
You're right about that.

**Pull request merged**